src/Core/src/Platform/Android/ViewExtensions.cs

```diff
@@ -198,17 +197,39 @@ internal static void UpdateBackground(this EditText platformView, IView view)
 				mauiDrawable.Dispose();
 			}
 
-			// Android will reset the padding when setting a Background drawable
+			if (platformView.Background is LayerDrawable layerDrawable)
+			{
+				platformView.Background = null;
+				layerDrawable.Dispose();
+			}
+
+			// Android will reset the padding when setting a Background drawable	
 			// So we need to reapply the padding after
 			var padLeft = platformView.PaddingLeft;
 			var padTop = platformView.PaddingTop;
 			var padRight = platformView.PaddingRight;
 			var padBottom = platformView.PaddingBottom;
 
-			var previousDrawable = platformView.Background;
-			var backgroundDrawable = paint!.ToDrawable(platformView.Context);
-			LayerDrawable layer = new LayerDrawable(new Drawable[] { backgroundDrawable!, previousDrawable! });
-			platformView.Background = layer;
+			var paint = view.Background;
+
+			Drawable? defaultBackgroundDrawable = ContextCompat.GetDrawable(platformView.Context, Resource.Drawable.abc_edit_text_material);
+
+			var previousDrawable = defaultBackgroundDrawable ?? platformView.Background;
+			var backgroundDrawable = paint.ToDrawable(platformView.Context);
+
+			if (previousDrawable is null)
+				platformView.Background = backgroundDrawable;
+			else
+			{
+				if (paint.IsNullOrEmpty())
```

Comment:
Can we change this to

`if (backgroundDrawable is null)` ?

If we do that then we can get rid of the null forgiveness operator on line 229